feature-utils/poly-analysis/src/report/report-analysis.js

```diff
@@ -0,0 +1,14 @@
+import RootAnalysis from "../../root-analysis.js";
```

Comment:
Maybe this whole directory structure should be flattened. I don't think it functionally requires all this. Having an "analysis" directory in a module called "analysis" does look a bit redundant.

Reply:
True, but if it causes lots of file changes I'd prefer to do that after this one lands, to keep this reviewable.